packages/block-editor/src/store/defaults.js

```diff
@@ -160,8 +160,10 @@ export const SETTINGS_DEFAULTS = {
 	// Allows to disable Openverse media category in the inserter.
 	enableOpenverseMediaCategory: true,
 
+	// Is the selected block cleared upon deselection.
```

Comment:
Maybe just a tiny rewording here? Or even remove it, since we have the description above in line 30?

Reply:
Good point, I've removed it 👍